src/it/java/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/LineLengthTest.java

```diff
@@ -46,4 +47,15 @@ public void testLineLength() throws Exception {
         verify(checkConfig, filePath, expected, warnList);
     }
 
+    @Test
+    public void testLineLengthJsniMethods() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+
+        final Configuration checkConfig = getModuleConfig("LineLength");
+        final String filePath = getPath("InputLineLengthJsniMethods.java");
```

Comment:
you need to load two modules from config. There examples already, please find them in code.